### PR TITLE
various changes to README

### PR DESCRIPTION
**Pull request by waldyrious**
- change main description
- wrap long lines
- include the word "translation" in the text
- change youtube link to point to the actual playlist page
- change the google groups links to the static versions
- remove redundant "i18n and l10n" in every bullet point entry
- add some missing links
- a few other wording tweaks

### CODE REVIEW

**Comment by waldyrious** [comment]
(Note: the rich diff is probably the easiest way to view this change)

File: README.md [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
+* The official Julia [documentation](http://docs.julialang.org/en/latest/),
+  [open books](https://en.wikibooks.org/wiki/Introducing_Julia) and manuals,
+  as well as other important sources of documentation
+  such as [*StackOverflow Documentation*](http://stackoverflow.com/documentation/julia-lang/topics),
+  [*Learn Julia in Y minutes*](https://learnxinyminutes.com/docs/julia/), and more,
+  whenever and however possible.
+* The documentation system in general,
+  including the [documentation of Julia packages](https://github.com/JuliaDocs/Documenter.jl/issues/297).
+* Transcripts, captions (subtitles), titles and descriptions for
+  [video lectures](https://www.youtube.com/user/JuliaLanguage)
```
Comment (Ismael-VC) [inline README.md]: There should be a comma between video lectures and video tutorials.

Comment (Ismael-VC) [reply README.md]: Everything else is excellent, after adding the comma, feel free to merge it! 👍

**Comment by Ismael-VC** [comment]
Maybe we should also reference the github projects, currently only:
- https://github.com/Julia-i18n/julia-i18n/projects/1

It's a more visual aid of the current status of Julia-i18n.

**Comment by waldyrious** [comment]
@Ismael-VC thanks for merging :) I was going to reply that the github projects link should be added in a different PR, since it would imply changing / removing the Roadmap section.

**Comment by Ismael-VC** [comment]
Yes, I agree with you, thanks again!

**Pull request merged**